### PR TITLE
Updated CHANGELOG.md

### PR DESCRIPTION
**Pull request by ok-reza**
Upon review, made a first round of changes to CHANGELOG to fix errors and adjust syntax. Also adjusted names where needed in order to match 'Community Features' language (for easy searchability).

(I made this a draft since I wasn't sure if I was making the pull request properly by using 'release/1.1' as branch and 'community' as base and didn't want to screw anything up. Please advise 😅 )

### CODE REVIEW

**Comment by sapphire-arches** [comment]
You want to use community as the base and target, we can tag it to cherry pick in to community.

**Comment by seangoodvibes** [comment]
It seems like you used a very old branch to update the change log because it's very hard to follow what changes you made. It's showing changes we made a long time ago.

I think you'll need to start over with a new PR which is based off the latest community build.

Also when updating the change log, please don't remove the c1.2.0 entries.

Thanks!
